rust-toolchain

```diff
@@ -1,2 +1,2 @@
 [toolchain]
-channel = "1.70.0"
+channel = "stable"
```

Comment:
Oops, didn't want to include this in this PR, I was curious to see if it build on stable (1.73.0), and it does.

I can remove if you think it's safer to stick to 1.70, or make the change in an other PR

Reply:
`stable` would force us to be pro-active on testing this project against new toolchains as soon as they are released. I like the idea, but we might not have this bandwidth.

Reply:
but it you tested with 1.73, let's upgrade!

Reply:
Yeah indeed it might mean code breaking for now apparent reason in the future.
I switched to 1.73 then